src/api/ledger/parse/account-order.js

```diff
@@ -1,14 +1,27 @@
 'use strict';
-const parseOrderBase = require('./order-base');
+const utils = require('./utils');
+const flags = utils.core.Remote.flags.offer;
+const parseAmount = require('./amount');
 
 // rippled 'account_offers' returns a different format for orders than 'tx'
```

Comment:
Right, the TransactionType:OfferCreate transaction Flags are different to the flags set in the LedgerEntryType:Offer Flags.